ls1intum/Artemis
# Remove router-outlet and isVisible definitions

### Checklist


### Motivation and Context

Follow-up to #2658.

### Description

This basically does the same as #2631. In Angular the parent component of a route will be rendered before the child component. To avoid this we either have to hide the parent component, or define a new route without a parent component. This PR changes all places using the former method to use the latter method.

### Steps for Testing

1. Make sure you can still view, create and edit Lectures and System Notifications
2. Make sure you can still view and edit Users
3. There should be no errors in the console and the breadcrumbs still work like on `develop`
